data/json/items/containers.json

```diff
@@ -970,7 +968,7 @@
         "rigid": true,
         "watertight": true,
         "open_container": true,
-        "sealed_data": { "spoil_multiplier": 1.1 },
+        "spoil_multiplier": 1.1,
```

Comment:
In this case, a "sealed" one will not spill because of open_container, but a non sealable one will always spill.

Reply:
I see. But I'm not sure why sealing it would make it spoil faster?

Reply:
possibly a typo?

Reply:
```suggestion
        "sealed_data": { "spoil_multiplier": 1.0 },
```